ateam_ai/src/behavior/behavior_evaluator.cpp

```diff
@@ -1,3 +1,22 @@
+// Copyright 2023 A Team
```

Comment:
Did we mean to have our copyright twice in a row in this file?

Reply:
Nope, I'll remove it